ui/apps/platform/src/services/ClustersService.ts

```diff
@@ -99,6 +99,7 @@ export function fetchClusterWithRetentionInformation(id: string): Promise<Cluste
 
 export type AutoUpgradeConfig = {
     enableAutoUpgrade?: boolean;
+    autoUpgradeAllowed?: 'NOT_ALLOWED' | 'ALLOWED';
```

Comment:
You can delete `?` if the response object always includes the properties.

Reply:
Done